res/values-pt-rBR/strings.xml

```diff
@@ -892,7 +892,7 @@
     <string name="ROOM">SALA</string>
     <string name="EXPERIMENTAL">EXPERIMENTAL</string>
     <string name="members">Membros</string>
-    <string name="Delete_ALL_Messages_">Excluir TODAS as Mensagens!</string>
+    <string name="Delete_ALL_Messages_">Excluir Todas as Mensagens!</string>
```

Comment:
Please do not remove all the capital letters here, because it's based on english value.